docker_ci/apt_installs.sh

```diff
@@ -32,3 +31,6 @@ apt-get install -y \
 
 # Clear cache
 rm -rf /var/lib/apt/lists/*
+
+# Set Python 3 as the default interpreter
+update-alternatives --install /usr/bin/python python /usr/bin/python3 10
```

Comment:
This is what I had to issue manually on my apt based installs. Looks good to me.